src/en/manhwa18/src/eu/kanade/tachiyomi/extension/en/manhwa18/Manhwa18.kt

```diff
@@ -1,62 +1,217 @@
 package eu.kanade.tachiyomi.extension.en.manhwa18
 
-import eu.kanade.tachiyomi.multisrc.mymangacms.MyMangaCMS
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
 import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.text.SimpleDateFormat
+import java.util.Locale
+import kotlin.math.min
 
-class Manhwa18 : MyMangaCMS("Manhwa18", "https://manhwa18.com", "en") {
+class Manhwa18 : HttpSource() {
+
+    override val baseUrl = "https://manhwa18.com"
+    private val apiUrl = "https://cdn3.manhwa18.com/api/v1"
+    override val lang = "en"
+    override val name = "Manhwa18"
+    override val supportsLatest = true
 
-    // Migrated from FMReader to MyMangaCMS.
     override val versionId = 2
```

Comment:
bump if urls have changed

Reply:
the manga/chapter url stay the same, no need to update version so user can continue using existed library entries without needs to migrate